src/xenia/kernel/util/game_info_database.cc

```diff
@@ -149,6 +149,87 @@ GameInfoDatabase::Achievement GameInfoDatabase::GetAchievement(
   return achievement;
 }
 
+GameInfoDatabase::PropertyBag GameInfoDatabase::GetPropertyBag(
+    xam::PropertyBag property_bag) const {
+  PropertyBag property_bag_native = {};
+
+  property_bag_native.contexts = std::set<uint32_t>(
```

Comment:
Can't it be just simple copy assignment? Input is a set and output is also a set